python/lsst/meas/base/baseMeasurement.py

```diff
@@ -137,7 +137,42 @@ def setupSchema(self, schema):
             aliases.set("slot_CalibFlux", self.calibFlux)
 
 
-class BaseMeasurementConfig(lsst.pex.config.Config):
+class SimpleBaseMeasurementConfig(lsst.pex.config.Config):
```

Comment:
Is the `Simple` deliberate because it has some connection to the `SimpleAlgorithm` super-class, or is this a coincidence?  If so a docstring clarifying that would be helpful.  (Or if not there's a confusing name clash here)

Reply:
No. This is something that Jim and I pair coded together to avoid having to have a huge refactor by changing the meaning of `BaseMeasurementTask`. As the note in `SimpleBaseMeasurementTask` says, it primarily exists to allow code to be shared between `SimpleForcedMeasurementTask` and `ForcedMeasurementTask`, where `SimpleForcedMeasurementTask` is a simplified version of forced measurement that assumes that uses a PSF-based footprint for every source, eliminating the need to transform the footprints. So this is really the workhorse of this ticket.

Reply:
Looking back at the code now I think the idea was that forced photometry on CCDs doesn't need all of the config options from `BaseMeasurementTask` so Jim just wanted to create a base task for the base task. If you have suggestions for a better naming convention I'm happy to hear it. We could also take it to `dm-naming-things`.